core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -132,7 +132,7 @@ class BrokerTopicMetrics(name: Option[String]) extends KafkaMetricsGroup {
   }
 }
 
-object BrokerTopicStats extends Logging {
+object BrokerTopicStats {
```

Comment:
The `BrokerTopicStats` refactoring causes the PR to be much larger than I think it needs to be. Can we move this change out into a separate cleanup PR?